### DISCUSSION TITLE
API ERROR

### DISCUSSION THREAD

**Discussion by udien-arrow** [kirimwa-id/forum#137] [Tanya Jawab] [answered]
Apakah Sekarang lagi error koneksinya?

Exception Object ( [message:protected] =>
503 Service Temporarily Unavailable
[string:Exception:private] => [code:protected] => 503 [file:protected] => C:\xampp\htdocs\kirimwa.id\function.php [line:protected] => 44 [trace:Exception:private] => Array ( [0] => Array ( [file] => C:\xampp\htdocs\kirimwa.id\quota.php [line] => 10 [function] => apiKirimWaRequest [args] => Array ( [0] => Array ( [token] => 6RkaKJU329b5_QIns_3e6dVo752ZqXuaxh15yegzp|CCD{VU [url] => https://api.kirimwa.id/v1/quotas ) ) ) ) [previous:Exception:private] => )

**Comment by udien-arrow** [comment, marked as answer]
Sudah bisa lagi dan lancar apinya sekarang, Pak @rioastamal, Btw ada group telegram atau wa tidak ya?
oia pak sekalian apakah api nanti juga mendukung button dan location?

**Comment by pst1812** [comment]
ketemu solusinya gimana ya kak? saya baru nemu problem yg sama

**Comment by yefridho** [comment]
Sepertinya ini karena masalah error server.... tiba2 error.... dan tiba2 bisa...
